packages/frontend/src/components/SchedulersView/LogsView.tsx

```diff
@@ -160,17 +140,18 @@ const Logs: FC<LogsProps> = ({
             {
                 id: 'jobs',
                 label: 'Job',
-                cell: (item) => {
-                    const currentLogs = getCurrentLogs(item, logs);
-                    const { handleLogs, sendLogs } =
-                        getFilteredLogs(currentLogs);
-                    return currentLogs.length > 0 ? (
+                cell: (item, currentLogs) => {
+                    return !currentLogs || currentLogs.length === 0 ? (
+                        <Text fz="xs" fw={500}>
+                            No jobs yet
+                        </Text>
+                    ) : (
                         <Stack spacing="md" fz="xs" fw={500}>
                             <Group spacing="two">
                                 <Text>All jobs</Text>
                                 <ActionIcon
                                     onClick={() =>
-                                        handleTogle(item.schedulerUuid)
+                                        handleTogle(currentLogs[0].jobGroup!)
```

Comment:
to avoid `!` which is unsafe operation in TS - we can have jobGroup passed as an argument to cell

`cell: (item, jobGroup, currentLogs)`

I will write in the review where you should pass it